.changelog/add-change.sh

```
@@ -0,0 +1,377 @@
+    if [[ "$where_i_am" =~ /.changelog$ || "$where_i_am" =~ /scripts$ ]]; then
+        # If this is in the .changelog or scripts directory, assume it's {repo_root}/<dir>.
+        repo_root="$( dirname "$where_i_am" )"
+    else
+        # Not in a git repo, and who knows where this script is in relation to the root,
+        # so let's just hope that our current location is the repo root.
+        repo_root='.'
+    fi
+fi
+[[ -n "$verbose" ]] && printf ' Repo root dir: [%s].\n' "$repo_root"
+
+valid_sections=( $( "${where_i_am}/get-valid-sections.sh" ) )
```

Shellcheck

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)